src/System.Text.Json/src/System/Text/Json/Serialization/Converters/JsonValueConverterChar.cs

```diff
@@ -21,20 +21,20 @@ public override bool TryRead(Type valueType, ref Utf8JsonReader reader, out char
             return true;
         }
 
-        public override void Write(char value, ref Utf8JsonWriter writer)
+        public override void Write(char value, Utf8JsonWriter writer)
         {
 #if BUILDING_INBOX_LIBRARY
-            Span<char> temp = MemoryMarshal.CreateSpan<char>(ref value, 1);
+            Span<char> temp = MemoryMarshal.CreateSpan(ref value, 1);
             writer.WriteStringValue(temp);
```

Comment:
Nit: you don't actually need a temp here; it can be a single line to parallel the other else case, e.g.
```C#
writer.WriteStringValue(
#if BUILDING_INBOX_LIBRARY
    MemoryMarshal.CreateSpan(ref value, 1)
#else
    value.ToString()
#endif
    );
```
or something like that.

Reply:
I decided to keep changes outside of the writer minimal, but that would work.

However, your suggestion does make it a bit harder to read (but VS coloring would help).

cc @steveharter